src/core/control/xojfile/LoadHandler.cpp

```diff
@@ -1045,7 +1063,7 @@ void LoadHandler::parserText(GMarkupParseContext* context, const gchar* text, gs
                     handler->stroke->setPressure(handler->pressureBuffer);
                 }
             } else {
-                g_warning("%s", FC(_F("xoj-File: {1}") % handler->filepath.string().c_str()));
+                g_warning("%s", FC(_F("xoj-File: {1}") % handler->filepath.u8string()));
```

Comment:
Thank you, then we at least know that g_error and siblings assume utf-8 but it seems to be that this is still bugged on windows. One example is this: https://gitlab.gnome.org/GNOME/glib/-/issues/3341 but we should use utf-8 in the future, because it's not intended to not support utf-8.